server/src/main/java/com/cloud/uuididentity/UUIDManagerImpl.java

```diff
@@ -119,7 +119,9 @@ public boolean IsUuidFormat(String uuid) {
         }
         Identity identity = (Identity) this._entityMgr.findById(entityType, customId);
         if (identity == null) {
-            throw new InvalidParameterValueException("Unable to find UUID for id " + customId);
+            throw new InvalidParameterValueException(String.format("Unable to find UUID for id %s of type %s",
```

Comment:
I would suggest you to use `[]` as var delimiter; IMHO, it is easier to recognize what is a variable and what is fixed in the final message; e.g.:

`String.format("Unable to find UUID for id [%s] of type [%s].",...`

Reply:
will do